opennms-provision/opennms-detector-simple/blueprint-detector.xml

```diff
@@ -15,7 +15,8 @@
 	<service id="icmpDetector" ref="icmp"
```

Comment:
yes, using them for getting the names in the Completer,  I can use the same factory there as well.